src/xjadeo/xjadeo.c

```diff
@@ -1396,7 +1403,11 @@ int open_movie (char* file_name) {
 	}
 
 	// Get a pointer to the codec context for the video stream
+#if LIBAVFORMAT_VERSION_INT < AV_VERSION_INT(57, 33, 100)
 	pCodecCtx=pFormatCtx->streams[videoStream]->codec;
+#else
+	pCodecCtx=pFormatCtx->streams[videoStream]->codecpar;
+#endif
```

Comment:
This seems wrong and will likely lead to crashes.

A pointer to the AVCodecContext is needed, while this patch assigns `AVCodecParameters*`.